recipes/gsort/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: gsort
+  version: '0.0.1'
+
+source:
+  fn: gsort_darwin_amd64 # [osx]
+  url: https://github.com/brentp/gsort/releases/download/v0.0.1/gsort_darwin_amd64 # [osx]
```

Comment:
@chapmanb I thought we recommend to build from source? Is this not true anymore or does the build time exceed the travis limit?